pkgs/applications/editors/010-editor/default.nix

```
@@ -0,0 +1,83 @@
+{ lib, stdenv, fetchFromGitHub, fetchurl, makeDesktopItem, autoPatchelfHook, copyDesktopItems, perlPackages, zlib, fontconfig, freetype, libxkbcommon, cups, ... }:
+
+let
+  extract = fetchFromGitHub {
+    owner = "lod";
+    repo = "unpack-install-jammer";
+    rev = "41faa5a8c7925db4ff7ca665cca478979d177de4";
+    sha256 = "1gvx7nkg4spz245hnmzcshiy0k3qvslqlzp4if7ngdqi2pfjrn4q";
+  };
+in
+stdenv.mkDerivation rec {
+  pname = "010-editor";
+  version = "12.0.1";
+
+  src = fetchurl {
+    url = "https://download.sweetscape.com/010EditorLinux64Installer${version}.tar.gz";
+    sha256 = "0ln8qxy1mij8qnnn8dxwnvmkx69l0f0w6xnb615ihx8wyhvlmfzh";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    copyDesktopItems
+  ] ++ (with perlPackages; [
+    perl
+    ModernPerl
+    CompressRawLzma
+    CompressRawZlib
+    FileHomeDir
+    TermProgressBar
+    DataDump
+  ]);
+
+  buildInputs = [
+    zlib
+    freetype
+    fontconfig
+    libxkbcommon
+    cups
+  ];
+
+  # Archive does not have a subdirectory
+  setSourceRoot = "sourceRoot=`pwd`";
```

```suggestion
  setSourceRoot = "sourceRoot=$PWD";
```

```suggestion
  setSourceRoot = "sourceRoot=$(pwd)";
```

Either is fine but we don't want to use deprecated back ticks